exercises.ipynb

```diff
@@ -133,39 +114,50 @@
    },
    "outputs": [],
    "source": [
-    "# Exercise #2\n",
-    "\n",
-    "# Now write code that gets you all the links from ALL the pages.\n",
-    "\n",
-    "# HINT: you will probably want to extract the URL in the \"next\" button on \n",
-    "# the bottom of the search pagination, which looks like \">\".\n",
+    "def find_next_page(soup):\n",
+    "    titles_sec = soup.find('div', attrs={'class': 'pagination'})\n",
+    "    next_page = titles_sec.find('li', attrs={'class': 'next'})\n",
+    "    next_page = next_page.find('a')\n",
```

Comment:
This is nicer with css notation, no? `'div.pagination li.next a'`, all in one line!